include/azure-tools.bash

```diff
@@ -0,0 +1,22 @@
+azure-configure-arm() {
+    declare desc="Configure new ARM application"
+
+    docker run -it sequenceiq/azure-cli-tools:0.5 arm_configure \
+      --app_name ${1:? app_name is required as 1. param} \
+      --app_password ${2:? app_password is required as 2. param} \
+      --subscription_id ${3:? subscription_id is required as 3. param} \
+      --username ${4:? username is required as 4. param} \
+      --password ${5:? password is required as 5. param}
+}
+
+azure-deploy-dash() {
+    declare desc="Deploy the MicrosoftDX/DASH project to a cloud service and create its storage accounts "
+
+    docker run -it sequenceiq/azure-cli-tools:0.5 deploy_dash \
+      --accounts ${1:? accounts is required as 1. param} \
+      --prefix ${2:? prefix is required as 2. param} \
+      --location ${3:? location is required as 3. param} \
+      --instances ${4:? instances is required as 4. param} \
+      --username ${5:? username is required as 5. param} \
+      --password ${6:? password is required as 6. param}
```

Comment:
The full parameter list could be passed to the docker container instead of getting the parameters one-by-one, so parameter handling won't have to be duplicated here and the command could be written as:
```
cbd azure deploy-dash --instances 3 --prefix dash ...
```
instead of having to add all the properties without switches in a pre-defined order